src/sentry/search/events/fields.py

```diff
@@ -2157,6 +2157,25 @@ def is_accessible(self, acl=None):
 FUNCTION_ALIAS_PATTERN = re.compile(r"^({}).*".format("|".join(list(FUNCTIONS.keys()))))
 
 
+def normalize_percentile_alias(args: Mapping[str, str]) -> Union[float, str, int]:
```

Comment:
```suggestion
def normalize_percentile_alias(args: Mapping[str, str]) -> str:
```

Reply:
> Nice, we should backlog something where we update this function to take the function instead of an alias, but this LGTM

Cool, added a jira ticket for the followup